packages/linux-drivers/mali-utgard/patches/add-meson-platform.patch

```diff
@@ -0,0 +1,301 @@
+diff --git a/driver/src/devicedrv/mali/platform/meson/meson.c b/driver/src/devicedrv/mali/platform/meson/meson.c
```

Comment:
Instead of a patch (to whole file) actual file could be used in package folder as _sources/driver/src/devicedrv/mali/platform/meson/meson.c_. It would be cleaner use in my oppinion.

Reply:
Good idea! I can do this for both supported platforms.

Reply:
Btw: You have similar patches in other commit(s) too.